### PR TITLE
Rework data export descriptions

### PR DESCRIPTION
**Pull request by noon-dawg**
## Summary

This pull request reworks data export descriptions, and related files, to conform to the [writing style](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md).

## Links

N/A

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/DataExports/DataExport.yaml [hunk 1]
```diff
@@ -41,68 +41,70 @@ properties:
...
+      For more information, see [Arguments](https://docs-reconciliation.rebilly.com/getting-started/arguments).
     type: object
     properties:
       filter:
-        description: >-
-          The collection items filter requires a special format. Use "," for
-          multiple allowed values. Use ";" for multiple fields.
+        description: |-
+          Filters the collection items. This field requires
+          a special format. Use `,` for multiple allowed values.  Use `;` for multiple fields.
```
Comment (ashkarpetin) [inline openapi/components/schemas/DataExports/DataExport.yaml:52]:
```suggestion
          a special format. Use `,` for multiple allowed values. Use `;` for multiple fields.
```

File: openapi/components/schemas/DataExports/DataExport.yaml [hunk 2]
```diff
@@ -41,68 +41,70 @@ properties:
...
+        description: |-
+          Sorts and orders the collection of items.
+          To sort in descending order, prefix with `-`.
       q:
         type: string
-        description: The partial search of the text fields.
+        description: Use this field to perform a partial search of text fields.
   emailNotification:
-    description: List of emails to be notified when export is completed.
+    description: List of email address to notify when an export is completed.
```
Comment (ashkarpetin) [inline openapi/components/schemas/DataExports/DataExport.yaml:66]:
```suggestion
    description: List of email addresses to notify when an export is completed.
```

File: openapi/components/schemas/DataExports/DataExportDateRange.yaml [hunk 3]
```diff
@@ -1,25 +1,25 @@
 type: object
-description: >-
-  The date range (can be in relative formats).  If omitted, all time will be
-  included. See the [Date Range
-  guide](https://docs-reconciliation.rebilly.com/getting-started/date-range)
-  for more options and examples.
+description: |-
+  Date range of the data export.
+  If this value is not set,
+  all data from the first recorded instance is included.
```
Comment (ashkarpetin) [inline openapi/components/schemas/DataExports/DataExportDateRange.yaml:5]: > all data from the first recorded instance is included

I am not clear what does that mean

File: openapi/components/schemas/DataExports/resources/customers.yaml [hunk 4]
```diff
@@ -1,4 +1,4 @@
-description: Customers resource type to export.
+description: Type of customers resource to export.
```
Comment (ashkarpetin) [inline openapi/components/schemas/DataExports/resources/customers.yaml:1]:
```suggestion
description: Customers resource to export.
```

That doesn't seem correct

File: openapi/components/schemas/DataExports/DataExportDateRange.yaml (hunk 3 above)
Comment (noon-dawg) [reply openapi/components/schemas/DataExports/DataExportDateRange.yaml]: Good point. I updated this to: `If this value is not set, all data is included.`

File: openapi/components/schemas/DataExports/resources/customers.yaml (hunk 4 above)
Comment (noon-dawg) [reply openapi/components/schemas/DataExports/resources/customers.yaml]: Updated to: `Customer resource to export.`

File: openapi/components/schemas/DataExports/resources/invoiceItems.yaml [hunk 5]
```diff
@@ -1,10 +1,10 @@
-description: Invoice items resource type to export.
+description: Type of invoice item resource to export.
```
Comment (ashkarpetin) [inline openapi/components/schemas/DataExports/resources/invoiceItems.yaml:1]:
```suggestion
description: Invoice item resource to export.
```

File: openapi/components/schemas/DataExports/resources/invoices.yaml [hunk 6]
```diff
@@ -1,4 +1,4 @@
-description: Invoices resource type to export.
+description: Type of invoice resource to export.
```
Comment (ashkarpetin) [inline openapi/components/schemas/DataExports/resources/invoices.yaml:1]:
```suggestion
description: Invoice resource to export.
```

File: openapi/components/schemas/DataExports/resources/revenueAudit.yaml [hunk 7]
```diff
@@ -1,4 +1,4 @@
-description: Revenue audit resource type to export.
+description: Type of revenue audit resource to export.
```
Comment (ashkarpetin) [inline openapi/components/schemas/DataExports/resources/revenueAudit.yaml:1]:
```suggestion
description: Revenue audit resource to export.
```

File: openapi/components/schemas/DataExports/resources/subscriptions.yaml [hunk 8]
```diff
@@ -1,4 +1,4 @@
-description: Orders resource type to export.
+description: Type of order resource to export.
```
Comment (ashkarpetin) [inline openapi/components/schemas/DataExports/resources/subscriptions.yaml:1]:
```suggestion
description: Order resource to export.
```

File: openapi/components/schemas/DataExports/resources/transactions.yaml [hunk 9]
```diff
@@ -1,10 +1,10 @@
-description: Transactions resource type to export.
+description: Type of transaction resource to export.
```
Comment (ashkarpetin) [inline openapi/components/schemas/DataExports/resources/transactions.yaml:1]:
```suggestion
description: Transaction resource to export.
```

**Review by ashkarpetin — APPROVED** [review]

**Review by n-fasano — APPROVED** [review]

**Pull request merged**